Resources/Prototypes/_Harmony/Entities/Mobs/Player/silicon.yml

```diff
@@ -1,3 +1,57 @@
+
+# Nuclear Operative Borgs
+- type: entity
+  id: BorgChassisGenericSaboteurBattery
+  parent: BorgChassisGenericSaboteur
+  suffix: Battery, Module, Operative
+  components:
+  - type: NukeOperative
+  - type: ContainerFill
+    containers:
+      borg_brain:
+      - PositronicBrain
+      borg_module:
+      - BorgModuleTool
+      - BorgModuleSyndicateWeapon
+  - type: ItemSlots
+    slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
+        startingItem: PowerCellHyper
```

Comment:
Description from the parent says: `A streamlined engineering cyborg, equipped with covert modules. Its chameleon projector lets it disguise itself as a Nanotrasen cyborg.`, which seems to be not true